gateway-provider-identity-assertion-common/src/main/java/org/apache/knox/gateway/identityasserter/common/filter/CommonIdentityAssertionFilter.java

```diff
@@ -55,9 +56,16 @@ public class CommonIdentityAssertionFilter extends AbstractIdentityAssertionFilt
 
   public static final String GROUP_PRINCIPAL_MAPPING = "group.principal.mapping";
   public static final String PRINCIPAL_MAPPING = "principal.mapping";
+  /* Service specific impersonation params that needs to be scrubbed */
+  public static final String IMPERSONATION_PARAMS = "impersonation.params";
```

Comment:
Can't we use the same constant here that is defined in the abstract class?

Reply:
@zeroflag which constant are you referring to? or are you suggesting moving this to `AbstractIdentityAssertionFilter`

Reply:
@smolnar82 there is already a constant like this in AbstractIdentityAsserterDeploymentContributor.java

Reply:
I see, I will change it, thanks!